back/apps/core-fca/src/dto/redirect-to-idp.dto.ts

```diff
@@ -10,4 +17,9 @@ export class RedirectToIdp extends CrsfToken {
   @IsOptional()
   @IsAscii()
   readonly identityProviderUid?: string;
+
+  @IsBoolean()
+  @IsOptional()
+  @Transform(({ value }) => value === 'on' || value === true)
```

Comment:
The value comes from a form on our side. I expect the transformation to have only one comparaison if any.